config/checkstyle_checks.xml

```diff
@@ -485,6 +485,7 @@
     <module name="NoLineWrap"/>
     <module name="NoWhitespaceAfter">
       <property name="tokens" value="ARRAY_INIT"/>
+      <property name="tokens" value="AT"/>
```

Comment:
please add this token to google_style too.
It is not clearly defined in style guide but, it is nonsense, most engineers even do not know that space is possible by syntax.

Reply:
As you add this to google config, please also add it to google's tests under `/src/it`.

Reply:
@romani and @rnveach in `google_check.xml `is no description module `NowhiteSpaceAfter`. In `/src/it `is no the tests for `NoWhitespaceAfter`. Are you sure what it need for current issue? I think what is task for new issue.

Reply:
No need to add it, fact that it is missed is unrelated issue.

Reply:
@romani Does this warrant a new issue to add it?